lib/exceptions.rb

```diff
@@ -170,6 +170,7 @@ class UpgradeFailed < MachineryError; end
     class BinaryDiffError < MachineryError; end
 
     class SshConnectionFailed < MachineryError; end
+    class SshConnectionDisrupted < SystemExit; end
```

Comment:
That's what we have the `MachineryError` exception class for, though. If you look at https://github.com/SUSE/machinery/blob/master/lib/cli.rb#L105 you'll see that these exceptions are handled properly and no backlog is shown.